script/testing/junit/traces/select.test

```diff
@@ -580,6 +580,55 @@ SELECT str_a_val NOT LIKE 'AbCdEf' AS result FROM like_tbl WHERE str_a_val IS NU
 statement ok
 DROP TABLE like_tbl;
 
+# index-join
+statement ok
+CREATE TABLE xxx (idx INT PRIMARY KEY, valx INT);
+
+statement ok
+CREATE TABLE yyy (idy INT PRIMARY KEY, valy INT);
+
+statement ok
+INSERT INTO xxx VALUES (3,3), (4,4), (5,5);
+
+statement ok
+INSERT INTO yyy VALUES (3, 3), (4,4);
+
+query I nosort
+SELECT * FROM xxx INNER JOIN yyy ON xxx.idx > yyy.idy;
```

Comment:
Thank you for the general tests! These generally look pretty good. For verifiability, can you also add a test that flips the two tables such that `xxx.idx` appears on the right side of the comparison as opposed to the left?

Reply:
> Thank you for the general tests! These generally look pretty good. For verifiability, can you also add a test that flips the two tables such that `xxx.idx` appears on the right side of the comparison as opposed to the left?

I‘ve added one test.